image/image.tf

```diff
@@ -1,4 +1,11 @@
+resource "libvirt_pool" "default" {
+  name = "default"
+  type = "dir"
+  path = "/var/lib/virt/pools/default"
+}
+
 resource "libvirt_volume" "ubuntu_focal_2022_12_13" {
   name   = "ubuntu-focal-2022-12-13"
+  pool   = libvirt_pool.default.name
```

Comment:
You can just put pool = 'default'

Reply:
After installing libvirt I had no pool so there was an error here. I searched in the official documentation and didn't find an explanation.

I found this though from https://gist.github.com/plembo/13ce29c9279807adfd9bd6b959f43fac:
> Some distributions create the defaut KVM (libvirtd) storage pool for images when they install KVM, others do this upon the creation of the first KVM guest.

I'm on Ubuntu. We could state in the README that this setup was tested on Ubuntu with success and it might not work for other distributions. What do you think?

Reply:
Awesome, thanks =)